test/spec/ViewerSpec.js

```diff
@@ -754,11 +754,11 @@ describe('Viewer', function() {
           var viewer = new Viewer({ container: container });
 
           // when
-          viewer.importXML(multipleXML, 'Diagram_2');
+          viewer.importXML(multipleXML, 'BpmnDiagram_2');
 
           // then
-          viewer.on('import.done', function(event) {
-            done();
+          viewer.on('import.done', function(context) {
```

Comment:
It is actually an event, not some context and I'd keep the _event_ identifier because of that.